images/libvirt-kubevirt/qemu-kube

```diff
@@ -93,24 +93,8 @@ set -e
 # to adhere to the resource limitations of the container.
 exec sudo -C 10000 unshare --mount bash -s << END
 
-  function _term() {
-    rm -rf /etc/resolv.conf.\$\$
-    pkill -P \$! --signal SIG\$1
-  }
-
-  trap "_term TERM" TERM
-  trap "_term INT" INT
-  trap "_term HUP" HUP
-  trap "_term QUIT" QUIT
-  trap "_term TERM" EXIT
-  trap "_term TERM" ERR
-
-  # Make sure qemu can resolve kubernetes DNS entries
-  nsenter -m -t $CONTAINER_PID cat /etc/resolv.conf > /etc/resolv.conf.\$\$
-  mount --bind /etc/resolv.conf.\$\$ /etc/resolv.conf
-
   cgclassify -g ${CGROUPS}:$CGROUP_PATH --sticky \$\$
-  nsenter -t $CONTAINER_PID -n -p $CMD &
+  nsenter -t $CONTAINER_PID -p $CMD &
```

Comment:
As this will break DNS resolving within the cluster, this will break test cases which require cluster level DNS resoltuion of the iscsi service.

However, once a patchset follows with moving libvirt's networking into the pod, then this should be fixed again.

@berrange @rmohr does it make sense to include a patch in this patchset actually to use pod networking for the libvirt pod - even if it break some other stuff, it should still allow us to keep some more tests running.